app/lib/time_limit_config.rb

```diff
@@ -49,6 +49,15 @@ def self.rules
       chase_candidate_before_dbd: [
         Rule.new(nil, nil, 5),
       ],
+      ucas_match_candidate_withdrawal_request: [
+        Rule.new(nil, nil, 10),
+      ],
+      ucas_match_candidate_withdrawal_request_reminder: [
+        Rule.new(nil, nil, 5),
+      ],
+      ucas_match_ucas_withdrawal_request: [
```

Comment:
When I was preparing the show and tell presentation realised I got the rules of the timings wrong 🤦‍♀️ 

It should be UCAS withdrawal request is sent 10 days after the initial email / 5 days after the reminder email / 5 days after candidate_last_contacted_at. 
We are currently doing 15 days after the initial email / 10 days after the initial email

I think this commit should do it https://github.com/DFE-Digital/apply-for-teacher-training/commit/b68fe077bbddf01bf60020960a2de3247019eea1
Feel free to cherry-cherry pick or I’m happy to raise a PR once yours is merged.